crates/web5/src/credentials/mod.rs

```diff
@@ -14,13 +14,14 @@ pub use status_list_credential::{
     STATUS_LIST_CREDENTIAL_TYPE,
 };
 pub mod verifiable_credential_1_1;
+pub mod verifiable_presentation_1_1;
```

Comment:
I think for both of these lines we should follow the same pattern of...

```suggestion
mod verifiable_credential_1_1;
mod verifiable_presentation_1_1;
```

and then subsequently follow-up with a `pub use` for the specific features which the APID makes available -- not strictly necessary at this time, but if not done here I'll probably follow-up with these proposed changes

Reply:
yup added